roles/edpm_download_cache/defaults/main.yml

```diff
@@ -26,3 +26,5 @@ edpm_download_cache_packages: true
 edpm_download_cache_container_images: true
 
 edpm_download_cache_running_services: "{{ edpm_services_override | default(edpm_services) }}"
+
+edpm_podman_auth_file: "/etc/containers/auth.json"
```

Comment:
this is a non-standard path. I'm curious why it's a better choice than `~/.config/containers/auth.json`, which would at least be checked when running podman commands manually, or if someone forgets to specify `--authfile`

while there is mention of `~/.config/containers/auth.json` being for Windows/MacOS, it's also checked on Linux according to my testing, and it is aligned with $XDG_CONFIG_HOME.

```
[root@edpm-compute-0 ~]# podman logout --authfile ~/.config/containers/auth.json registry.redhat.io
Removed login credentials for registry.redhat.io
[root@edpm-compute-0 ~]# podman logout registry.redhat.io
Error: not logged into registry.redhat.io
[root@edpm-compute-0 ~]# podman pull registry.redhat.io/rhoso/openstack-ovn-controller-rhel9@sha256:5e8e082f30f876e67797ded5e9f34ac2c66e82149e3c4407f4f47e6affbcb217
Trying to pull registry.redhat.io/rhoso/openstack-ovn-controller-rhel9@sha256:5e8e082f30f876e67797ded5e9f34ac2c66e82149e3c4407f4f47e6affbcb217...
Error: initializing source docker://registry.redhat.io/rhoso/openstack-ovn-controller-rhel9@sha256:5e8e082f30f876e67797ded5e9f34ac2c66e82149e3c4407f4f47e6affbcb217: unable to retrieve auth token: invalid username/password: unauthorized: Please login to the Red Hat Registry using your Customer Portal credentials. Further instructions can be found here: https://access.redhat.com/RegistryAuthentication
[root@edpm-compute-0 ~]# podman login --authfile ~/.config/containers/auth.json registry.redhat.io -u jslagle@redhat.com
Password: 
Login Succeeded!
[root@edpm-compute-0 ~]# podman pull registry.redhat.io/rhoso/openstack-ovn-controller-rhel9@sha256:5e8e082f30f876e67797ded5e9f34ac2c66e82149e3c4407f4f47e6affbcb217
Trying to pull registry.redhat.io/rhoso/openstack-ovn-controller-rhel9@sha256:5e8e082f30f876e67797ded5e9f34ac2c66e82149e3c4407f4f47e6affbcb217...
Getting image source signatures
Checking if image destination supports signatures
Copying blob 7ea6a9f79cdd skipped: already exists  
Copying blob c25b9b426d72 skipped: already exists  
Copying blob 8c519c115fa8 skipped: already exists  
Copying blob 03f7ea57e338 skipped: already exists  
Copying config dcc1786b8f done   | 
Writing manifest to image destination
Storing signatures
dcc1786b8fc5c7d096fc70e7da2db2356b2cfee401e3924de14788b2f96ccdb1
[root@edpm-compute-0 ~]# 
```

notice the last `podman pull` works without having to specify `--authfile`.

Reply:
Switched to `~/.config/containers/auth.json`